.gitignore

```diff
@@ -1,34 +0,0 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-**/node_modules
-**/.pnp
-.pnp.js
-
-# testing
```

Comment:
Now you decided to delete it ? :d

Reply:
Did it delete the entire .gitignore? I'm just speechless :|